README.md

```diff
@@ -9,17 +9,43 @@ Free and guided workshops in Rust – a systems programming language that runs b
 Our events are aimed at people who are underrepresented in technology and offer the opportunity to learn within a friendly, safe and sometimes fun environment. We do this by providing rewarding learning experiences, building helpful tools as well as materials and gathering a group of awesome humans. RustBridge is a project by the Rust Language Community Team.
 
 # Requirments 
-Rust development enviroment setup: [Rust](https://www.rust-lang.org/en-US/)
-PostgreSQL database setup: [PostgreSQL](https://www.postgresql.org/)
+* Rust development enviroment setup: [Rust](https://www.rust-lang.org/en-US/)
+* PostgreSQL database driver: [Diesel](http://diesel.rs/guides/getting-started/)
+* PostgreSQL database setup: [PostgreSQL](https://www.postgresql.org/)
+
+*Note*: some of the dependencies used in this project use nightly builds. You may need to use nightly to run the project. To install nightly run: `rustup install nightly` to use nightly run: `rustup run nightly rustc`, and to set default to nightly: `rustup default nightly`
```

Comment:
"You may need to use nightly" i think you definitely need to use nightly to run this, the note is good i would just make it stronger by saying "you *will* need to use nightly"